src/mame/drivers/raiden2.cpp

```diff
@@ -1477,24 +1061,44 @@ MACHINE_CONFIG_DERIVED(raiden2_state::raidendx, raiden2)
 MACHINE_CONFIG_END
 
 
+MACHINE_CONFIG_START(raiden2_state::zeroteam_sound)
+
+	/* basic machine hardware */
+	MCFG_CPU_ADD("audiocpu", Z80, SOUND_CLOCK)
+	MCFG_CPU_PROGRAM_MAP(zeroteam_sound_map)
+
+	/* sound hardware */
+	MCFG_SPEAKER_STANDARD_MONO("mono")
+
+	MCFG_SOUND_ADD("ymsnd", YM3812, SOUND_CLOCK)
+	MCFG_YM3812_IRQ_HANDLER(DEVWRITELINE("seibu_sound", seibu_sound_device, fm_irqhandler))
+	MCFG_SOUND_ROUTE(ALL_OUTPUTS, "mono", 1.0)
+
+	MCFG_OKIM6295_ADD("oki", OKI_CLOCK, PIN7_HIGH)
+	MCFG_SOUND_ROUTE(ALL_OUTPUTS, "mono", 0.40)
+
+	MCFG_DEVICE_ADD("seibu_sound", SEIBU_SOUND, 0)
+	MCFG_SEIBU_SOUND_CPU("audiocpu")
+	MCFG_SEIBU_SOUND_ROMBANK("seibu_bank1")
+	MCFG_SEIBU_SOUND_YM_READ_CB(DEVREAD8("ymsnd", ym3812_device, read))
+	MCFG_SEIBU_SOUND_YM_WRITE_CB(DEVWRITE8("ymsnd", ym3812_device, write))
+MACHINE_CONFIG_END
+
 MACHINE_CONFIG_START(raiden2_state::zeroteam)
 
 	/* basic machine hardware */
-	MCFG_CPU_ADD("maincpu", V30,XTAL(32'000'000)/2) /* verified on pcb */
+	MCFG_CPU_ADD("maincpu", V30, MAIN_CLOCK) /* verified on pcb */
 	MCFG_CPU_PROGRAM_MAP(zeroteam_mem)
 	MCFG_CPU_VBLANK_INT_DRIVER("screen", raiden2_state,  raiden2_interrupt)
 
 	MCFG_MACHINE_RESET_OVERRIDE(raiden2_state,zeroteam)
 
-	MCFG_CPU_ADD("audiocpu", Z80, XTAL(28'636'363)/8)
-	MCFG_CPU_PROGRAM_MAP(zeroteam_sound_map)
-
 	/* video hardware */
 	MCFG_SCREEN_ADD("screen", RASTER)
 	MCFG_SCREEN_VIDEO_ATTRIBUTES(VIDEO_UPDATE_AFTER_VBLANK)
 //  MCFG_SCREEN_REFRESH_RATE(55.47)    /* verified on pcb */
-	MCFG_SCREEN_RAW_PARAMS(XTAL(32'000'000)/4,512,0,40*8,282,0,32*8) /* hand-tuned to match ~55.47 */
```

Comment:
Why have you removed the comment?  Have you actually measured screen timing on a real PCB?